common/libs/validators.js

```diff
@@ -143,5 +143,5 @@ export function isValidRawTx(rawTx: RawTransaction): boolean {
 // Full length deterministic wallet paths from BIP32
 // https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
 export function isValidPath(dPath: string) {
-  return dPath.split('\'/').length === 4;
+  return dPath.split("'/").length === 4;
```

Comment:
All in favor of this, but why didn't prettier wallop you?

Reply:
This was actually prettier

Reply:
Hmm, I'll have to figure out why mine was forcing me into using single quotes